### PR TITLE
feat: revamp InputFee

### PR DESCRIPTION
**Pull request by alesmit**
## Summary

https://app.clickup.com/t/m35nw6

## Screenshot

image

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] Tests
-   [x] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
Task linked: [CU-m35nw6 [Send Page] New Transaction Fee Design](https://app.clickup.com/t/m35nw6)

File: src/domains/transaction/components/InputFee/InputFeeSimple.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+						{loading ? (
+							<Skeleton width={100} className="h-3 my-1" />
+						) : (
+							<>
+								<div className="font-normal text-sm">
+									{optionValue} {ticker}
+								</div>
+								{showConvertedValues && (
+									<div className="font-normal text-sm text-theme-secondary-500 mt-3">
+										<Amount normalize ticker={exchangeTicker} value={valueConverted} />
```
Comment (w3ea) [inline src/domains/transaction/components/InputFee/InputFeeSimple.tsx]:
```suggestion
										<Amount normalize={false} ticker={exchangeTicker} value={valueConverted} />
```
Screenshot 2021-06-11 184213
Screenshot 2021-06-11 183926

Comment (w3ea) [reply src/domains/transaction/components/InputFee/InputFeeSimple.tsx]: BTW, nice job :ok_hand:

Comment (alesmit) [reply src/domains/transaction/components/InputFee/InputFeeSimple.tsx]: Updated! Thanks @w3ea

Comment (w3ea) [reply src/domains/transaction/components/InputFee/InputFeeSimple.tsx]: i think remove `normalize`, doesn't fix it
i didn't check it
https://github.com/ArkEcosystem/desktop-wallet/blob/develop/src/app/components/Amount/Amount.tsx#L60

Comment (alesmit) [reply src/domains/transaction/components/InputFee/InputFeeSimple.tsx]: Great catch. I set that as false. Ty

**Review by faustbrian — APPROVED** [review]

**Pull request merged**